### ISSUE TITLE
Maintenance status ?

### ISSUE THREAD

**Issue by rzr** [sandeepmistry/node-bluetooth-hci-socket#123]
Hi,

Is there any interest to keep maintaining this software ? 

If yes please review/merge or pick changes from:
https://github.com/noble/node-bluetooth-hci-socket/pull/122

If not a "community fork" is maintained elsewhere:

https://abandonware.github.io/ 

Relate-to: abandonware/abandonware.github.io#5 Regards

**Comment by rzr** [comment]
Our latest release is:

https://www.npmjs.com/package/@abandonware/bluetooth-hci-socket/v/0.5.3-4

**Comment by rzr** [comment]
reviews welcome:

https://github.com/noble/node-bluetooth-hci-socket/pull/122

**Comment by rzr** [comment]
FYI an updated version has been released:

https://www.npmjs.com/package/@abandonware/bluetooth-hci-socket/v/0.5.3-5

**Comment by rzr** [comment]
update: New downstream release:

https://www.npmjs.com/package/@abandonware/bluetooth-hci-socket/v/0.5.3-8

Relate-to: https://github.com/abandonware/node-bluetooth-hci-socket/issues/30

**Comment by ebaauw** [comment]
> Is there any interest to keep maintaining this software ?

Yes, please.  Currently using the downstream fork by abandonware.

**Comment by rzr** [comment]
upstream feedback welcome,

meanwhile free to use our latest release:

https://www.npmjs.com/package/@abandonware/bluetooth-hci-socket/v/0.5.3-12
